### ISSUE TITLE
[ART] Updated: Team Sunflower Vs Team Goblin

### ISSUE THREAD

**Issue by rissojavier** [sunflower-land/sunflower-land#548]
- # **Team Sunflower**
Team Sunflower:
![Team Sunflowers](https://user-images.githubusercontent.com/88810285/163921076-c670f614-e4e3-41fb-9ba7-89f4b865a9ad.gif)

Menu icon:
![icon-menu-ts](https://user-images.githubusercontent.com/88810285/163921086-e3209c47-a2ec-4578-a299-acb064477565.png)

Capture:
![zoom-ts](https://user-images.githubusercontent.com/88810285/163921610-7be1135b-58a5-4543-beee-34c310d619ba.gif)

- #  **Team Goblin**
Team Goblin:
![Team-Goblin](https://user-images.githubusercontent.com/88810285/163921134-2c2a2641-78a3-4e3b-a649-8496942d876f.gif)

Menu icon:
![icon-menu-tg](https://user-images.githubusercontent.com/88810285/163921140-1c5e6996-3840-4f5e-99ef-e813eec9ab5e.png)

Capture:
![zoom-tg](https://user-images.githubusercontent.com/88810285/163921643-81fc4c69-b5bf-41aa-b727-d279f31cbb52.gif)

> **Polygon Wallet:** 0x5D338b5AFa38A9e53a69e2D6458CdA693A6BB214
> **Farm ID:** 10242

**Comment by mecsafonso** [comment]
Holy cuacks, the normal one looks so good, kinda expensive but I love it!!!

**Comment by JacobSobolev** [comment]
I'm almost finished implementing this.

I have a suggestion about the ingredients and the effects.
first, let's convert everything into SFL (based on NFT army spreadsheet)
T1 Water Tank: 
- 50 Parsnip = 25 SFL (0.5 per a seed)
- 100 wood = 26 SFL (around 0.26 per wood)
- 10 SLF
total: ~61

now let's move to the T2 Tank:
- T1 Water Tank: 61 SFL
- 150 Iron: 259.5 SFL ( 1.73 per Iron)
- 75 SFL
total: 395

now that we know roughly their equivalent to SFL we can compare it to other items.
for example, the carrot sword cost around 210 and gives 300% mutant crops.

because we have one item that gives the mutant crops buff I was thinking to give those items some other buff related to the water. like increasing the fishing rewards for the T1 by 50% and for the T2 by 300%.
Or maybe combine it with another proposal I have seen related to the water like on the [Pond Revamp](https://github.com/sunflower-land/sunflower-land/issues/456) maybe increase the rarity spawn rate of an Axolotl

@rissojavier let me know what you think.
also would appreciate others' feedback on this suggestion with an upvote or a downvote

**Comment by rissojavier** [comment]
It seems to me very well thought @jacobsobolev, for my part I have only proposed the cost of materials and buffs as a little detailed idea, that is, I dedicated myself to design the pixel art with a concept and did not do the ROI calculations or something like that.

@jacobsobolev for my part you have the free way to propose the buffs and the cost of manufacturing this art.
Traducido con DeepL https://www.deepl.com/app/?utm_medium=android-share

**Comment by wendin017** [comment]
omg, I really liked it, would love to have on my farm <3

**Comment by Netherzapdos** [comment]
Hey everyone! Saw that my Axolotl proposal was mentioned here yesterday but forgot to reply 😅 First of all, love the art and congratz on getting it in the game ^^

I see that you guys wanted to think of a utility for the tanks and thought that maybe increasing an Axolotl spawn can be its utility. I've actually thought about making a statue that provides such buffs already 😅 however, I spent a good time thinking of another buff that has the same idea.

Basically, each of these statues would introduce a unique type of Axolotl that will only spawn when you have these. For example, the bird bath would increase the spawn rate of a "Farmer Axoltol" (just an idea 🤣 ) or a "Goblin Axolotl".

Unsure if you guys have already thought of something, but figured I'd give my thoughts as well :) @JacobSobolev @rissojavier

**Comment by rissojavier** [comment]
> Hey everyone! Saw that my Axolotl proposal was mentioned here yesterday but forgot to reply 😅 First of all, love the art and congratz on getting it in the game ^^
> 
> I see that you guys wanted to think of a utility for the tanks and thought that maybe increasing an Axolotl spawn can be its utility. I've actually thought about making a statue that provides such buffs already 😅 however, I spent a good time thinking of another buff that has the same idea.
> 
> Basically, each of these statues would introduce a unique type of Axolotl that will only spawn when you have these. For example, the bird bath would increase the spawn rate of a "Farmer Axoltol" (just an idea 🤣 ) or a "Goblin Axolotl".
> 
> Unsure if you guys have already thought of something, but figured I'd give my thoughts as well :) @JacobSobolev @rissojavier 

I think it's a good idea

**Comment by JacobSobolev** [comment]
@Netherzapdos sounds great, for now, it acts as a decorative NFT.
maybe once your proposal got implemented it can interact with those NFTs

**Comment by Netherzapdos** [comment]
@JacobSobolev Yeah the Axolotl proposal might take a while to get reviewed though 😅 So if you guys have already thought of a utility then don't wait for my proposal and just go ahead 😄

**Comment by Cicciolin0** [comment]
Team Goblin deserves It's own pool! and it looks so cute with it's green blob and a goblin in It!

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
